services/web/server/src/simcore_service_webserver/director_v2/_models.py

```diff
@@ -75,13 +77,13 @@ class Config(BaseCluster.Config):
 
 
 class ClusterPatch(BaseCluster):
-    name: str | None
+    name: str | None  # type: ignore[assignment]
     description: str | None
-    type: ClusterType | None
-    owner: GroupID | None
+    type: ClusterType | None  # type: ignore[assignment]
+    owner: GroupID | None  # type: ignore[assignment]
```

Comment:
what is mypy complaining here? what the assignment stand for?

Reply:
it's because it is overwriting the type from a base classe. Since the type is being changed it complains